LaunchSpatial.bat

```diff
@@ -0,0 +1,13 @@
+@echo off
```

Comment:
Cool, should probably put this in StarterProject and TestSuite too if it's not already there

Reply:
They've already got this, it failed to make it's way into TPS when it was in limbo.